openreview/conference/matching.py

```diff
@@ -605,6 +613,26 @@ def _build_subject_area_scores(self, submissions):
             raise openreview.OpenReviewException('Failed during bulk post of edges! Scores found: {0}, Edges posted: {1}'.format(len(edges), edges_posted))
         return invitation
 
+    def _compute_scores(self, score_invitation_id, submissions, matching_status):
+        invitation = self._create_edge_invitation(score_invitation_id)
+
+        job_id = self.client.request_expertise(name=self.conference.get_short_name(), group_id=self.match_group.id, paper_invitation=self.conference.get_blind_submission_id(), model='specter+mfr')
```

Comment:
I think we need to add the expertise selection invitation to send to the API to ignore the excluded papers.